umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/dif10.clj

```diff
@@ -24,6 +24,7 @@
    [cmr.umm-spec.versioning :as umm-spec-versioning]
    [cmr.umm.dif.date-util :refer [parse-dif-end-date]]))
 
+;; TODO how do we know which values are supported for this version?
```

Comment:
Remove the TODO, there is only 1 DIF 10 version that we support. It is dif-10.2.